examples/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -172,7 +172,7 @@ message ABitOfEverything {
 	repeated Nested nested = 2;
 	float float_value = 3 [(grpc.gateway.protoc_gen_swagger.options.openapiv2_field) = {description: "Float value field", default: "0.2", required: ['float_value']}];
 	double double_value = 4;
-	int64 int64_value = 5;
+	int64 int64_value = 5 [(grpc.gateway.protoc_gen_swagger.options.openapiv2_field) = {type: INTEGER}];;
```

Comment:
Could we add another int64 field instead of overwriting the properties of the existing one? We still want to test that it's a string by default.

Reply:
Sure, I added another one.